Dockerfile

```diff
@@ -49,7 +49,10 @@ RUN apt-get update \
 
 COPY package.json package-lock.json /usr/src/app/
 
-RUN JOBS=MAX npm ci --no-optional --unsafe-perm || JOBS=MAX npm install --no-optional --unsafe-perm
+# install the newer NPM so we can use npm ci
+RUN npm install -g npm@6
```

Comment:
You can move this before the `COPY package.json package-lock.json /usr/src/app/` to increase caching